backend/src/main/scala/petclinic/Main.scala

```diff
@@ -5,9 +5,25 @@ import petclinic.services._
 import zio._
 import zio.logging.backend.SLF4J
 import zio.logging.removeDefaultLoggers
+import zio.metrics.connectors.{MetricsConfig, newrelic}
 
+/** Main is the entry point for the application.
+  *
+  * Because it extends ZIOAppDefault, we are required to define a `run` method
+  * in which our side-effecting logic is housed. `run` is responsible for
```

Comment:
I wonder if "side-effecting" is helpful here. It presupposes a good amount of functional programming knowledge. I wonder if "logic", "application logic", or "main program logic" would be more accessible.

Reply:
That's a valid point. Maybe eventually it would be worth going the route of (re)adding higher-level descriptions but supporting them with links to explanatory documentation. But, I think you're right and I went the route of "application logic".